deploy/terraform-custom-datacommons/modules/terraform.tfvars.sample

```diff
@@ -22,3 +22,6 @@ project_id  = "your-gcp-project"
 
 # User provided namespace. Example: "dev", "prod", "dan-dev"
 namespace   = "john-doe-dev"
+
+# Data Commons API key. See https://docs.datacommons.org/api/rest/v1/getting_started#getting-api-keys
```

Comment:
Er, not really; should be https://docs.datacommons.org/custom_dc/quickstart.html#prerequisites

Reply:
Updated